src/state/loanReport/actions.js

```diff
@@ -52,35 +52,35 @@ export function updateLoanReport(payload) {
 }
 
 /**
- * Create an action to set current pair.
- * @param {string[]} pairs pairs
+ * Create an action to set current symbols.
+ * @param {string[]} symbols symbols
  */
-export function setTargetPairs(pairs) {
+export function setTargetSymbols(symbols) {
   return {
-    type: types.SET_PAIRS,
-    payload: pairs,
+    type: types.SET_SYMBOLS,
+    payload: symbols,
   }
 }
 
 /**
- * Create an action to add target pair.
- * @param {string} pair pair
+ * Create an action to add target symbol.
+ * @param {string} symbol symbol
```

Comment:
same here, we can remove `symbol` as description in JSDOC as it's the same as name of param